ruby/item.rb

```diff
@@ -23,7 +23,7 @@ def move_to_archive
 
   def add_label(label)
```

Comment:
The syntax for setter methods is `add_label=(label)`.
Kindly include the missing equal sign on the method definition.